privacyidea/api/periodictask.py

```diff
@@ -125,7 +137,10 @@ def set_periodic_task_api():
     active = is_true(getParam(param, "active", default=True))
     interval = getParam(param, "interval", optional=False)
     node_string = getParam(param, "nodes", optional=False)
-    node_list = [node.strip() for node in node_string.split(",")]
+    if node_string.strip():
+        node_list = [node.strip() for node in node_string.split(",")]
+    else:
+        raise ParameterError(u"nodes: expected at least one node, got {!r}".format(node_string))
```

Comment:
isn't `node_string` empty in the `else` case?

Reply:
Changed this in cb40d19